### ISSUE TITLE
showing failed node_exporter & blackbox_exporter in "tiup cluster display" list

### ISSUE THREAD

**Issue by tshqin** [pingcap/tiup#607]
pls add a feature, to show failed node_exporter & blackbox_exporter in "tiup cluster display" list, for the exporters in Up status, pls hide them as is.

**Comment by july2993** [comment]
@lucklove currently only when first time deploy these twe components will try to start it?

**Comment by lucklove** [comment]
@july2993 No. Every time the user runs `tiup cluster start` or `tiup cluster restart`

**Comment by july2993** [comment]
@tshqin the origin problem is after the machine restarted, start the cluster, but found exporter is not started later?
